exchange/options.go

```diff
@@ -66,7 +66,8 @@ type Options struct {
 	// least frequently used content is evicted to make more room for new content.
 	// Default is 10GB.
 	Capacity uint64
-
+	// PrivKey is the path of the private key the user wants to import
+	PrivKey string
```

Comment:
The exchange should not be concerned with a private key, the wallet module only should be concerned with key operations.

Reply:
The problem is that we are currently setting an address at 2 different places :
- in the exchange, when we generate a new key ([here](https://github.com/myelnet/pop/blob/main/exchange/exchange.go#L74))
- in the node when we import a key ([here](https://github.com/myelnet/pop/blob/main/node/popn.go#L233))

So when we import a key, we first generate one, then import it => We have 2 keys.
If you don't like the PrivKey in the exchange maybe instead we can handle the key generation/importation elsewhere? But then some tests will fail because they will expect a key being generated when doing an exchange.New(...)

Reply:
@gallexis that's a good point, let's move key generation out of the exchange so it's only handled at the node level

Reply:
I commit the changes but for some reason some tests are broken